AmplifyPlugins/API/Tests/AWSAPIPluginTests/Operation/GraphQLMutateCombineTests.swift

```diff
@@ -118,7 +118,7 @@ class GraphQLMutateCombineTests: OperationTestBase {
             }
         })
 
-        waitForExpectations(timeout: 0.05)
+        waitForExpectations(timeout: 1.0)
```

Comment:
Changed to 1.0 because this test running on specifically on watchOS was timing out.